test2/2.2/main.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+#include <locale.h>
+#include <string.h>
+#include "stack.h"
+
+int main() {
+    setlocale(LC_ALL, "RU-ru");
+    Node *stack = NULL;
+    while (true) {
+        printf("введите команду:\n");
+        char buffer[10];
+        scanf("%9s", buffer);
+        if (strcmp(buffer, "exit") == 0) {
+            deleteAll(&stack);
+            return 0;
+        }
+        if (strcmp(buffer, "push") == 0) {
+            int number = 0;
+            scanf("%d", &number);
+            push(&stack, number);
```

Comment:
```suggestion
            push(&stack, number);
            continue;
```
Если команда распознана и выполнена, то дальнейшая обработка команд не нужна.
При такой реализации при указании команды `push 10`, я получаю
```
введите команду:
push 10
неизвестная команда
```
Такая же проблема для остальных команд.
(-1 балл)